test/composables/useSidebar.multipleTags.test.ts

```
@@ -0,0 +1,233 @@
+import { describe, it, expect } from 'vitest'
```

ESLint

[error] Expected "expect" to come before "it".

(perfectionist/sort-named-imports)